frontend/src/metabase/setup/tests/common.unit.spec.tsx

```diff
@@ -164,20 +126,75 @@ describe("setup (OSS)", () => {
         expectSectionToHaveLabel("Add your data", "4");
         expectSectionToHaveLabel("Usage data preferences", "5");
       });
+    });
+  });
 
-      it("should not set the flag for the embedding homepage", async () => {
-        await setupForUsageQuestion();
-        selectUsageReason("self-service-analytics");
-        clickNextStep();
+  describe("embedding homepage flags", () => {
+    it("should set the correct flags when interested in embedding", async () => {
+      await setup();
+      skipWelcomeScreen();
+      skipLanguageStep();
+      await submitUserInfoStep();
+
+      selectUsageReason("embedding");
+      clickNextStep();
+
+      screen.getByText("Finish").click();
+
+      expect(await getLastSettingsPutPayload()).toMatchObject({
```

Comment:
since all of these use `toMatchObject` which I believe does partial matching, it doesn't guarantee that `setup-license-active-at-setup` won't be set. Should we use `toEqual` for all tests instead?

Reply:
Also done in 882ed72b4a5583f2610ea2b683ee4309021cb52f